saleor/graphql/order/tests/test_order.py

```diff
@@ -215,6 +216,88 @@ def test_order_query(
     assert expected_method.type.upper() == method["type"]
 
 
+ORDERS_WITH_INVOICES_QUERY = """
```

Comment:
What do you think about moving your tests to separate test file? Like `test_order_invoices` or something like this? This file has over 3k lines. It is really impossible to maintain tests in such huge file.